packages/l10n/src/utils.ts

```diff
@@ -1,4 +1,13 @@
-export const mapLocaleToIntlLocale = locale => {
+interface Currency {
+  label: string;
+  symbol: string;
+}
+
+interface Currencies {
+  [key: string]: Currency;
```

Comment:
is this right?: 🤔

Reply:
Yeah looks right to me.

Reply:
We can use a record: `Record<string, Currency>`